src/asymmetric_crypto_ctx.cpp

```diff
@@ -133,7 +123,7 @@ namespace mococrw {
                 throw MoCOCrWException("Expected RSA Key for RSA operation");
             }
         }
-
+    protected:
```

Comment:
This library prefixes class members with _. When you do a cleanup anyways,  you can just adapt it here as well.

Reply:
Done